src/main/java/picard/illumina/ExtractIlluminaBarcodes.java

```diff
@@ -371,9 +371,9 @@ protected int doWork() {
     public static void finalizeMetrics(final Map<String, BarcodeMetric> barcodeToMetrics,
                                        final BarcodeMetric noMatchMetric) {
         // Finish metrics tallying.
-        int totalReads = noMatchMetric.READS;
-        int totalPfReads = noMatchMetric.PF_READS;
-        int totalPfReadsAssigned = 0;
```

Comment:
I assume you'll want to change `BarcodeMetric` to store `long` instead of `int` as well?  One or two samples on a NovaSeq high-throughput flowcell would causethis.

Reply:
Hi @nh13 , 
Good point.  Will do.

-Alec